cmd/native-btc/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"os"
+
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/joho/godotenv"
+)
+
+// ENV variables
+const (
+	Host = "BTC_RPC"
+	User = "USER_RPC"
+	Pass = "PASS_RPC"
+)
+
+func main() {
+	err := godotenv.Load("../../.env")
```

Comment:
Do a check here if the file exists, if not then error that `.env` file missing on path `/path/to/env/file`

Reply:
Done